apps/arweave/src/ar_data_sync.erl

```diff
@@ -3493,7 +3555,7 @@ process_disk_pool_immature_chunk_offset(Iterator, TXRoot, TXPath, AbsoluteOffset
 		false ->
 			{Offset, _, ChunkSize, DataRoot, DataPathHash, ChunkDataKey, Key, _, _, _} = Args,
 			case update_chunks_index({AbsoluteOffset, Offset, ChunkDataKey,
-					TXRoot, DataRoot, TXPath, ChunkSize, unpacked}, State) of
+					TXRoot, DataRoot, TXPath, ChunkSize, unpacked}, false, State) of
```

Comment:
Why don't we add the footprint here? Is it because we're in the diskpool and only want to track footprints for persisted storage modules?

Reply:
Yes, we need to support removing everything above the given offset for the disk pool data (ar_sync_record:cut), which is expensive for footprints because we need to potentially iterate over many footprints removing the data from each.